### PR TITLE
gnome 43 compatibility

### PR DESCRIPTION
**Pull request by confuzled**
untested but should work

### CODE REVIEW

**Comment by confuzled** [comment]
it follows a similar pattern to your gnome 42 update, so I assume it would work, but I haven't tested it as I'm away, will test asap

**Comment by Cj-Malone** [comment]
Could you please test it for me? I'm not on 43 yet.

Copy the source into `${XDG_DATA_HOME:=$HOME/.local/share}/gnome-shell/extensions/fuzzy-clock@keepawayfromfire.co.uk/`
You probably have to restart GNOME or fully logout if on Wayland.

File: metadata.json [hunk 1]
```diff
@@ -14,9 +14,10 @@
 		"40.0",
 		"41.0",
 		"42.0"
```
Comment (Cj-Malone) [inline metadata.json]:
```suggestion
		"42.0",
```

**Comment by confuzled** [comment]
Yup, will do as a soon as i can, also accidentally marked the requested change as resolved without resolving it, but i will fix that as soon as i can

**Comment by confuzled** [comment]
my gnome 43 install is playing up a bit with extensions, so I will keep trying but It might take a while :/

**Comment by Cj-Malone** [comment]
No worries, I should get it fairly soon and will test then.

**Comment by sinisterra** [comment]
image

Tried it on my machine, the patch is working correctly on GNOME 43.

**Comment by Cj-Malone** [comment]
That's good enough for me, thank you both.

**Pull request merged**